CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ### Fixed
 
+- Public: Fix 422 error on snapshot upload
```

Comment:
Maybe we should also specify here what error is seen on the UI side to be clearer about what it fixes. 
I think it was always the "File type not supported" error looking at your findings in the ticket & the Zendesk ticket where this was also apparently happening.

Reply:
Definitely, good suggestion